src/ts/jsoneditor.ts

```diff
@@ -81,21 +85,25 @@ $(document).ready(function() {
     const saveJsonFile = function(){
       if (typeof currentFileItemID === 'undefined') {
         // we are creating a new file
-        const realName = prompt('Enter name of the file');
+        let realName = prompt('Enter name of the file');
```

Comment:
again, this should be i18n, but this is out of the scope of this PR. Note that the same string is used in doodle.ts.